### ISSUE TITLE
browser-sync not working with mamp pro

### ISSUE THREAD

**Issue by nricca** [BrowserSync/grunt-browser-sync#80]
Hi everyone,

i'm very excited to use browser-sync but unfortunately i cannot make it work with mamp pro.

My webiste is accessible via the url : http://fundeego.dev:8888 but nothing is diplayed via the localhost:3000, the name is not resolved and via the external url, the page is displayed but it took forever to load the content.

Here is my conf :
```js
    browserSync: {
            default_options: {
                bsFiles: {
                    src: [
                        "app/assets/css/*.css'",
                        "app/*.html"
                    ]
                },
                options: {
                    watchTask: true,
                    proxy: "http://fundeego.dev:8888/"
                }
            }
        }
```

Here is the hosts file edited by mamp pro : 

    127.0.0.1	Riccas-MacBook-Pro.home	# MAMP PRO - Do NOT remove this entry!
    ::1		Riccas-MacBook-Pro.home	# MAMP PRO - Do NOT remove this entry!
    fe80::1%lo0	Riccas-MacBook-Pro.home	# MAMP PRO - Do NOT remove this entry!
    127.0.0.1	fundeego.dev	# MAMP PRO - Do NOT remove this entry!
    ::1		fundeego.dev	# MAMP PRO - Do NOT remove this entry!
    fe80::1%lo0	fundeego.dev	# MAMP PRO - Do NOT remove this entry!

 Thanks for your help

**Comment by sollenne** [comment]
+1 for this issue. Any luck getting it working like this @nricca ?

**Comment by nricca** [comment]
Nope, didn't work with both options.
It's work very well outside of mamp, so I go back with livereload for this specific projet.
Thanks anyway

**Comment by danielkoch** [comment]
Are there any news on this issue?

**Comment by jimlakey** [comment]
why was this issue closed? It does not seem to have been resolved, unless I'm missing something. I'm having the same trouble with the latest version of BS.

Any help would be appreciated!

**Comment by shakyShane** [comment]
@jimlakey we'll happily reopen the ticket if someone could be kind enough to share a sample that demonstrates the issue :)

**Comment by jimlakey** [comment]
I'd be glad to, but you'll have to spell it out for me I'm afraid. What do I need to provide in a sample to show the issue?

**Comment by shakyShane** [comment]
Reproduce the bug in as little code as possible (could be an simple php file, or an entire framework) - include how you use BrowserSync (gulp, grunt, cli etc) - basically just create a repo that I can clone and see the issue. also as much Mamp config info as possible would be great.

**Comment by jimlakey** [comment]
I started building out a test case and came across the issue I was encountering while doing it. The issue I had was that I was trying to run `grunt browserSync` with `watchTask: true` set in the config. I then figured I'd run `grunt watch` afterwards. Instead I made by default grunt task `grunt.registerTask('default',   ['browserSync', 'watch']);` and then just run `grunt` on the command line and now it all works fine.

So I wonder if other people had a similar issue with the `watchTask: true` flag set but then not running the watch task at the same time, or in the wrong order. I'd read http://www.browsersync.io/docs/grunt/#grunt-watch but I didn't see that as the issue at the time!
